app/src/main/java/com/metro/gwuexplorer/MainActivity.kt

```diff
@@ -56,14 +60,34 @@ class MainActivity : AppCompatActivity() {
         
         go.setOnClickListener {
 
-            val choices = listOf("667 M Street Washington, DC 20585", "668 M Street Washington, DC 20585")
+           // val choices = listOf("667 M Street Washington, DC 20585", "668 M Street Washington, DC 20585")
+
+            // Pass a context (e.g. Activity) and locale
+            val geocoder = Geocoder(this, Locale.getDefault())
+            val locationName = stationname.getText().toString()
+
+            val maxResults = 2
+
+            val results: List<Address> = geocoder.getFromLocationName(locationName,maxResults)
```

Comment:
Make sure you come back and put this inside a `doAsync { ... }` like we learning in class to minimize the risk of getting an Application Not Responding (ANR) error.